ingest/validation/validate_metadata.py

```diff
@@ -553,6 +591,11 @@ def encode_term_iri(term_id, base_uri):
     encoded_iri = encoder.quote_plus(encoder.quote_plus(query_uri))
     return encoded_iri
 
+def extract_terminal_pathname(url):
+    """Extract the last path segment from a URL
+    """
+    return list(filter(None, url.split("/"))).pop()
```

Comment:
the rails URI gem might have this built in?

Reply:
Except that this is Python...